src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -47,20 +52,29 @@ export class WelcomePageContribution implements IWorkbenchContribution {
 			const hasBackups = await this.workingCopyBackupService.hasBackups();
 			if (hasBackups) { return; }
 
-			// Open the welcome even if we opened a set of default editors
-			if (!this.editorService.activeEditor || this.layoutService.openedDefaultEditors) {
-				const startupEditorSetting = this.configurationService.inspect<string>(configurationKey);
+			// Ensure Welcome page for first-launch, no matter what is open.
+			// FIXME: Logic inspired by contrib/welcome/telemetryOptOut/browser/telemetryOptOut.ts.
+			// FIXME: Migrate telemetryOptOutUrl to boolean telemetryOptOut
+			if (this.productService.telemetryOptOutUrl && !this.storageService.get(telemetryOptOutStorageKey, StorageScope.GLOBAL)) {
```

Comment:
If I immediately went to the welcome page from the getting started I would not get the telemetry footer.
I found this surprising, as I think we should show this until the user clicks on it at least?

Reply:
The goal is to make this more of an informative notice that should be visible on first open but doesn't require interaction (in contrast to the current sticky notification).